coder_sniffer/Drupal/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -94,13 +101,24 @@ public function process(File $phpcsFile, $stackPtr)
             break;
         }
 
-        // Constructor methods are exempt from requiring a docblock.
-        // @see https://www.drupal.org/project/coder/issues/3400560.
         $methodName = $phpcsFile->getDeclarationName($stackPtr);
-        if ($methodName === '__construct'
-            && $tokens[$commentEnd]['code'] !== T_DOC_COMMENT_CLOSE_TAG
+        if ($tokens[$commentEnd]['code'] !== T_DOC_COMMENT_CLOSE_TAG
             && $tokens[$commentEnd]['code'] !== T_COMMENT
         ) {
+            if ($methodName === '__construct') {
```

Comment:
we must not treat constructors in a special way. Comments on constructors are good and we need to allow them.